test/unit/test_utils.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 from PIL import Image
 
-from label_maker.utils import url, class_match, get_tile_tif
+from label_maker.utils import url, class_match, get_tile_tif, get_tile_wms
```

Comment:
Want to update the test for image fetching by testing `get_image_function` instead of `get_tile_tif`?

Reply:
The function to imagery string matching isn't ideal right now so I'd like it to remain untested for a bit (the tile "getter" functions themselves are the most important things to be reliable)